test/e2e/replica_set_arbiter/replica_set_arbiter_test.go

```diff
@@ -27,14 +27,29 @@ func TestReplicaSetArbiter(t *testing.T) {
 	ctx := setup.Setup(t)
 	defer ctx.Teardown()
 
+	// First valid case
+	numberArbiters := 1
+	numberMembers := 2
+	mdb, user := e2eutil.NewTestMongoDB(ctx, "mdb2", "")
+	mdb.Spec.Arbiters = numberArbiters
+	mdb.Spec.Members = numberMembers
+	pwd, err := setup.GeneratePasswordForUser(ctx, user, "")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Run("Create MongoDB Resource", mongodbtests.CreateMongoDBResource(&mdb, ctx))
+	t.Run("Check that the stateful set becomes ready", mongodbtests.StatefulSetBecomesReady(&mdb, wait.Timeout(20*time.Minute)))
+	t.Run("Check the number of arbiters", mongodbtests.AutomationConfigReplicaSetsHaveExpectedArbiters(&mdb, numberArbiters))
```

Comment:
We can check here if the MongoDB has reached `Running` phase -- since the issue mentions about the resource not reaching running state.

Reply:
Good point! Thanks for spotting this!

Reply:
I decided to rewrite the whole test and introduce a parametrized tests there. Should be more readable now.